etc/column.conf

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+# Log file
+log_file = /var/log/column.log
+
+# Log info level
+log_level = DEBUG
+
+[API]
+server = 127.0.0.1
+port = 5000
```

Comment:
Don't use 5000.  It's keystone's port and probably some others.  Find a better unused port.